src/Rheologies/elasto_visco_plastic_rheology.jl

```diff
@@ -25,6 +25,7 @@ struct ElastoViscoPlasticRheology{FT, IP}
         new{FT, IP}(P, C, e, Δ_min, α⁻, α⁺, c, ip)
 end
 
+struct ModifiedReplacementPressure end
```

Comment:
Is this the case with the max in the denominator? Is there a way to have a more descriptive name?